pyproject.toml

```diff
@@ -7,12 +7,14 @@ license = "CC BY-SA"
 
 [tool.poetry.dependencies]
 python = "^3.8"
-sphinx = "4.3.1"
-livereload = "2.6.3"
-sphinx-book-theme = "0.1.7"
-sphinx-copybutton = "0.4.0"
+
+sphinx = "*"
+livereload = "*"
+sphinx-book-theme = "*"
+sphinx-copybutton = "*"
```

Comment:
Hard pins don't work as then `poetry update` cannot do anything.

Some people might argue that we should have `"^4.3.1"` instead of `"*"`, because "that carries more information", but over time, this information gets more and more incorrect. Incorrect documentation is worse than no documentation.